deploy/test-environments/main.tf

```diff
@@ -10,7 +10,7 @@ locals {
     project  = "${var.project}"
     owner    = "${var.owner}"
   }
-  ec_url = "https://cloud.elastic.co"
+  ec_url = "${var.ec_url}"
```

Comment:
`var.ec_url` is used in the `restapi` provider and in the `ec_project` module. In both cases, this variable is used directly. So, there is no need to create` local.ec_url`, and it can be removed.